pkgs/top-level/all-packages.nix

```diff
@@ -4967,6 +4967,7 @@ in
 
   pinentry_ncurses = res.pinentry.override {
     gtk2 = null;
+    gcr = null;
```

Comment:
Can you move this to `pinentry` definition so that it applies to all the backends?